releasenotes/notes/fix-ad-annotated-service-41807217fcb061f8.yaml

```diff
@@ -0,0 +1,3 @@
+fixes:
+  - |
+    Fix issue preventing newly annotated services to be considered by autodiscovery in the cluster-agent
```

Comment:
This should be under `releasenotes-dca/notes`

Reply:
Yep indeed!